js/apps/system/_admin/aardvark/APP/react/public/index.html

```diff
@@ -10,6 +10,18 @@
     <meta name="description" content="ArangoDB Admin Web Interface">
     <meta name="author" content="Heiko Kernbach, Michael Hackstein">
     <script src="%REACT_APP_ARANGODB_CONFIG_PATH%"></script>
+
+    <script>
```

Comment:
see https://github.com/WICG/resize-observer/issues/38

Reply:
Maybe also worth adding this link: https://github.com/souporserious/react-measure/issues/104#issuecomment-438743015 

🙂

Reply:
the issue is: a dependecy of react-select is not using rAF, which it should; but there's no discussion about it in react-select library. 

This comment in the WICG issue says: "This error is often benign"  (https://github.com/WICG/resize-observer/issues/38#issuecomment-422126006).

In this case there's no impact, hence we are ignoring it

Note: I can track it to this lib: https://github.com/floating-ui/floating-ui/issues/1740#issuecomment-1156169497, where they don't want to use rAF because it causes a flicker, and instead suggest setting a max-width in react-select. If that works, will raise a PR there :) 



I